crates/rocket-cookie-auth/src/lib.rs

```diff
@@ -5,6 +5,7 @@ use anyhow::Result;
 use rocket::async_trait;
 
 pub mod auth;
+pub mod basic;
```

Comment:
🙃 We could rename the crate to `rocket-auth` and have modules for `basic` and `cookie`-based authentication.

```suggestion
pub mod cookie;
pub mod basic;
```